packages/experimental/Avatar/SPEC.md

```diff
@@ -0,0 +1,311 @@
+   */
+  src?: string;
+}
+```
+
+### Styling Tokens
+
+Tokens can be used to customize the styling of the control by using the `customize` function on the `Avatar`. For more information on using the `customize` API, please see [this page](../../framework/composition/README.md). The `Avatar` has the following tokens:
+
+```ts
+export interface JSAvatarTokens extends IBackgroundColorTokens, IForegroundColorTokens, AvatarConfigurableProps, IBorderTokens, FontTokens {
+  /**
+   * Avatar ocality which is changed depending on `active` prop.
+   */
+  avatarOpacity?: number;
+
+  /**
+   * The size of presence badge.
+   */
+  badgeSize?: BadgeSize;
+  /**
+   * The icon color.
+   */
+  iconColor?: ColorValue;
+
+  /**
+   * The size of the icon.
+   */
+  iconSize?: number;
+  /**
+   * The height of the avatar.
+   */
+  height?: number;
+
+  /**
+   * Avatar shapes:
+   */
+  circular?: JSAvatarTokens;
+  square?: JSAvatarTokens;
+
+  /**
+   * Token for inactive value of `active` prop
+   */
+  inactive?: JSAvatarTokens;
+
+  /**
+   * The width of the avatar.
+   */
+  width?: number;
+
+  /**
+   * Avatar sizes:
+   */
+  size20?: JSAvatarTokens;
+  size24?: JSAvatarTokens;
+  size28?: JSAvatarTokens;
+  size32?: JSAvatarTokens;
+  size36?: JSAvatarTokens;
+  size40?: JSAvatarTokens;
+  size48?: JSAvatarTokens;
+  size56?: JSAvatarTokens;
+  size64?: JSAvatarTokens;
+  size72?: JSAvatarTokens;
+  size96?: JSAvatarTokens;
+  size120?: JSAvatarTokens;
+
+  /**
+   * Avatar colors:
+   */
+  neutral?: JSAvatarTokens;
+  brand?: JSAvatarTokens;
+  darkRed?: JSAvatarTokens;
+  cranberry?: JSAvatarTokens;
+  red?: JSAvatarTokens;
+  pumpkin?: JSAvatarTokens;
+  peach?: JSAvatarTokens;
+  marigold?: JSAvatarTokens;
+  gold?: JSAvatarTokens;
+  brass?: JSAvatarTokens;
+  brown?: JSAvatarTokens;
+  forest?: JSAvatarTokens;
+  seafoam?: JSAvatarTokens;
+  darkGreen?: JSAvatarTokens;
+  lightTeal?: JSAvatarTokens;
+  teal?: JSAvatarTokens;
+  steel?: JSAvatarTokens;
+  blue?: JSAvatarTokens;
+  royalBlue?: JSAvatarTokens;
+  cornflower?: JSAvatarTokens;
+  navy?: JSAvatarTokens;
+  lavender?: JSAvatarTokens;
+  purple?: JSAvatarTokens;
+  grape?: JSAvatarTokens;
+  lilac?: JSAvatarTokens;
+  pink?: JSAvatarTokens;
+  magenta?: JSAvatarTokens;
+  plum?: JSAvatarTokens;
+  beige?: JSAvatarTokens;
+  mink?: JSAvatarTokens;
+  platinum?: JSAvatarTokens;
+  anchor?: JSAvatarTokens;
+}
+```
+
+## Behaviors
+
+### States
+
+The following section describes the different states which `Avatar` can be in as a result of interaction.
+
+#### showRing state
+
+Ring is shown when `showRing` is set to true
+
+#### transparentRing state
+
+Makes ring transparent
+
+#### showBadge state
+
+showBadge is set to true when `active` prop is `unset` or undefined
+
+### Interaction
+
+For now we won't add props for interactions.
+For the next step we will need to add tooltip and onClick handler.
+
+## Accessibility
```

Comment:
Will there be more information added here later?

Reply:
Yes. I haven't started working on accessibility task yet